cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -163,7 +181,9 @@ def index_item(item, skip_index=False):
                 item_index['id'] = item_id
                 if item.start:
                     item_index['start_date'] = item.start
-
+                item_index['content_groups'] = getattr(item, 'content_groups', None)
+                if item_index['content_groups'] == []:
+                    item_index['content_groups'] = None
```

Comment:
I've actually set it to None because these values get stored differently in a different searcher backend. Searching with this value set to None will return everything that has this field undefined/None, but not ones that have content groups defined to empty array/list.